packages/logger/src/Logger.ts

```diff
@@ -69,48 +105,120 @@ class Logger implements ClassThatLogs {
     });
   }
 
+  /**
+   * it adds the given attributes (key-value pairs) to all log items generated by this Logger instance.
+   *
+   * @param {LogAttributes} attributes
+   * @returns {void}
+   */
   public addPersistentLogAttributes(attributes?: LogAttributes): void {
     this.persistentLogAttributes = merge(attributes, this.getPersistentLogAttributes());
   }
 
+  /**
+   * Alias for addPersistentLogAttributes.
+   *
+   * @param {LogAttributes} attributes
+   * @returns {void}
+   */
   public appendKeys(attributes?: LogAttributes): void {
     this.addPersistentLogAttributes(attributes);
   }
 
+  /**
+   * it creates a separate Logger instance, identical to the current one.
+   * It's possible to overwrite the new instance options by passing them.
+   *
+   * @param {LoggerOptions} options
+   * @returns {Logger}
+   */
   public createChild(options: LoggerOptions = {}): Logger {
     return cloneDeep(this).setOptions(options);
   }
 
+  /**
+   * It prints a log item with level DEBUG.
+   *
+   * @param {LogItemMessage} input
+   * @param {Error | LogAttributes | unknown} extraInput
+   * @returns {void}
+   */
   public debug(input: LogItemMessage, ...extraInput: LogItemExtraInput): void {
     this.processLogItem('DEBUG', input, extraInput);
   }
 
+  /**
+   * It prints a log item with level ERROR.
+   *
+   * @param {LogItemMessage} input
+   * @param {Error | LogAttributes | unknown} extraInput
+   * @returns {void}
+   */
   public error(input: LogItemMessage, ...extraInput: LogItemExtraInput): void {
     this.processLogItem('ERROR', input, extraInput);
   }
 
+  /**
+   * It evaluates whether the current Lambda function invocation has a cold start or not.
+   *
+   * @static
+   * @returns {void}
+   */
   public static evaluateColdStartOnce(): void {
     if (!Logger.getColdStartEvaluatedValue()) {
       Logger.evaluateColdStart();
     }
   }
 
+  /**
+   * It returns a boolean value which is true if the current Lambda function cold start has been already evaluated, false if not.
```

Comment:
```suggestion
   * It returns a boolean value which is true if the current Lambda function cold start has been already evaluated, false otherwise.
```